wasm/legend/legendTxTypes/addLiquidityTypes.go

```diff
@@ -86,7 +86,7 @@ func ConstructAddLiquidityTxInfo(sk *PrivateKey, segmentStr string) (txInfo *Add
 	// compute call data hash
 	hFunc := mimc.NewMiMC()
 	// compute msg hash
-	msgHash, err := ComputeAddLiquidityMsgHash(txInfo, hFunc)
+	msgHash, err := txInfo.ComputeMsgHash(hFunc)
```

Comment:
`ComputeMsgHash` is a bit long,  what about `Hash`?

Reply:
Fixed